### ISSUE TITLE
Probleme bei Eventerstellung

### ISSUE THREAD

**Issue by leoselig** [hpi-swt2/event-und-raumplanung#322]
- Start-/Endzeit runden auf 15-Minuten-Intervalle
- Start-/Endzeit sollte initial nicht in Vergangenheit liegen
- "Schedule" nicht übersetzt
- Schedule-Feld sollte gleichen Style wie andere Inputs haben
- Impotant Flag raus?

**Comment by leoselig** [comment]
@rosabraatz

**Comment by cktrum** [comment]
Soll die Checkbox für 'Important' raus oder nicht?

**Comment by leoselig** [comment]
Ja, Checkbox raus und die important visualisierung generell auch :)
